src/core/Table/TableRowMemoized.tsx

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { Row, TableRowProps, TableState } from 'react-table';
+import { useIntersection } from '../utils/hooks/useIntersection';
+import { getCellStyle } from './utils';
+
+const TableRow = <T extends Record<string, unknown>>(props: {
```

Comment:
Description for this would be useful. 
Why we use memoized row, what deps (checks) should be added in future, etc...

Reply:
Added